packages/components/src/internal/components/samples/SampleAliquotAssaysCount.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, memo, useEffect, useMemo } from 'react';
+
+import {
+    InjectedAssayModel,
+    isLoading,
+    LoadingSpinner,
+    getSampleAssayQueryConfigs,
+    SchemaQuery,
+} from "../../..";
+
+// These need to be direct imports from files to avoid circular dependencies in index.ts
+import { InjectedQueryModels, withQueryModels } from '../../../public/QueryModel/withQueryModels';
+import { withAssayModels } from '../assay/withAssayModels';
+
+const SampleAliquotAssaysCountBodyImpl: FC<InjectedQueryModels> = memo(props => {
+    const { actions, queryModels } = props;
+    const allModels = Object.values(queryModels);
+    const allLoaded = allModels.every(model => !model.isLoading);
+
+    useEffect(() => {
+        actions.loadAllModels(true);
+    }, []);
```

Comment:
I believe we could do away with this and just put `autoLoad={true}` as a prop in `SampleAliquotAssaysCountImpl` next to where it passes the queryConfigs to this component.